flow/packet-headers/snmpv2.yaml

```diff
@@ -0,0 +1,201 @@
+components:
+  schemas:
+    Flow.Snmpv2:
+      description: SNMPv2 packet header
```

Comment:
If we do v3 later , would we need to repeat everything there ? Or can v2 definition be used with v3 as a superset on top of that ?

Reply:
Discussed